mathics/builtin/algebra.py

```diff
@@ -312,6 +332,148 @@ def apply(self, expr, evaluation):
         return from_sympy(result)
 
 
+class FactorTermsList(Builtin):
+    """
+    <dl>
+    <dt>'FactorTermsList[poly]'
+        <dd>returns a list in which the first element is the overall numerical factor in $poly$,
+        and the second element is the polynomial with the overall factor removed.
+    <dt>'FactorTermsList[poly,{x1, x2, ...}]'
+        <dd>returns a list of factors of $poly$.
+        The first element in the list is the overall numerical factor.
+        The second element is a factor that does not depend on any of the $xi$.
+        Subsequent elements are factors which depend on progressively more of the $xi$.
+    </dl>
```

Comment:
Can we rewrite these descriptions a little so they're not verbatim identical to MMA: https://reference.wolfram.com/language/ref/FactorTermsList.html